envoy.dependency.check/envoy/dependency/check/abstract/checker.py

```diff
@@ -195,6 +200,25 @@ async def dep_date_check(
                 "release_dates",
                 [f"Date matches ({dep.release_date}): {dep.id}"])
 
+    async def dep_sha_check(
+            self,
+            dep: "abstract.ADependency") -> None:
+        """Check sha for dependency."""
+        if not await dep.release.sha:
+            self.error(
+                "sha",
+                [f"{dep.id} is a GitHub repository with no inferrable "
+                 "release sha"])
+        elif await dep.release_sha_mismatch:
+            self.error(
+                "sha",
+                [f"SHA mismatch: {dep.id} "
+                 f"{dep.release_sha} != {await dep.release.sha}"])
+        else:
+            self.succeed(
+                "sha",
+                [f"SHA matches ({dep.release_sha}): {dep.id}"])
```

Comment:
in the code check code ive started using a tick for success:

https://github.com/envoyproxy/pytooling/blob/816be6adbb7fe7c29ad6ccfe9a8e39a2305f0da4/envoy.code.check/envoy/code/check/abstract/checker.py#L149-L152

i think we should replace `SHA matches` with `🗸 matches`